packages/core/src/payment/strategies/braintree/braintree-venmo-payment-strategy.ts

```diff
@@ -93,9 +94,18 @@ export default class BraintreeVenmoPaymentStrategy implements PaymentStrategy {
             throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
         }
 
+        const features = this._store.getState().config.getStoreConfigOrThrow()
+            .checkoutSettings.features;
+        const isBraintreeVenmoWebFallbackSupport = isExperimentEnabled(
+            features,
+            'PAYPAL-5406.braintree_venmo_web_fallback_support',
+        );
+
         try {
             this._braintreePaymentProcessor.initialize(clientToken);
-            this._braintreeVenmoCheckout = await this._braintreePaymentProcessor.getVenmoCheckout();
+            this._braintreeVenmoCheckout = await this._braintreePaymentProcessor.getVenmoCheckout({
+                mobileWebFallBack: isBraintreeVenmoWebFallbackSupport,
```

Comment:
So when experiment is disabled `mobileWebFallBack` will not be supported?
Or maybe in this case we should not provide any config to getVenmoCheckout? WDYT?

Reply:
Yes, will not. I added exp here to be able to turn off if something bad happened. We have some problem with venmo testing, I want to be sure that this field will not lead to specific behaviour and in this case we will be able to fix it by turning off this exp. If everything work as expected and problem related to this changes will be fixed I will remove related code and pass this parameter to where we set up this config. So lets leave it as it is.